src/structure/spatial.jl

```diff
@@ -187,6 +187,23 @@ function omegaangle(res::AbstractResidue, res_prev::AbstractResidue)
 end
 
 """
+Calculate the omega angle in radians for a residue in a `Chain`. Arguments are
+the chain and the first residue index (where atoms "N" and "CA" are taken
+from). The next residue is at pos + 1 (where atom "CA" and "C" iares required).
+"""
+function omegaangle(chain::Chain, pos::Int64)
+    if !(string(pos) in keys(chain.residues))
```

Comment:
Can use `haskey` and the defined residues accessor: `if !haskey(residues(chain), string(pos))`.